Compile warning fix

AlbertHungGarmin opened the pull request:
These pull requests contain the compiler warnings I encountered during my work.

AlbertHungGarmin committed:
Increase buffer for max string size

The maximum integer output for %d in sprintf ranges from 2 to 12 bytes,
including the terminating null character. Sprintf might write a
terminating null character beyond the end of an 11-byte buffer.
Enlarging the buffer to 12 bytes would be safer.

tencent-adm commented:
[![CLA assistant check](https://cla.opensource.tencent.com/pull/badge/not_signed)](https://cla.opensource.tencent.com/Tencent/rapidjson?pullRequest=2132) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.opensource.tencent.com/Tencent/rapidjson?pullRequest=2132) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.opensource.tencent.com/check/Tencent/rapidjson?pullRequest=2132) it.</sub>

AlbertHungGarmin committed:
Eliminate missing prototypes warning

Add prototype of testJson in cursorstreamwrappertest.cpp to remove
missing-prototypes or missing-declarations warning.
